transfer/transfer.process.protocol.md

```diff
@@ -22,36 +23,43 @@ This document outlines the key elements of the [Transfer Process Protocol](../mo
+A [Transfer Process](../model/terminology.md#transfer-process) (TP) involves two parties, a [Provider](../model/terminology.md#provider) that offers one or more [Datasets](../model/terminology.md#dataset) under a [Usage Policy](../model/terminology.md#policy) and a [Consumer](../model/terminology.md#consumer) that requests [Datasets](../model/terminology.md#dataset). A TP progresses through a series of states, which are controlled by the [Provider](../model/terminology.md#provider) and [Consumer](../model/terminology.md#consumer) using messages. A TP transitions to another state as a result of an exchanged message.
 
-A TP is managed by a [Connector](../model/terminology.md#connector--data-service-). The [Connector](../model/terminology.md#connector--data-service-) serves as a coordinating technical entity that receives counter-party messages and manages its local state of the TP. It may as well also operate the hosting of the [Datasets](../model/terminology.md#dataset), or control their offering through another system.
+### 1.1 Prerequisites
 
-### 1.1 Control and Data Planes
-A TP is managed by a [Connector](../model/terminology.md#connector--data-service-). The [Connector](../model/terminology.md#connector--data-service-) consists of two logical components, a _Control Plane_ and a _Data Plane_. The Control Plane serves as a coordinating layer that receives counter-party messages and manages the TP state. The Data Plane performs the actual transfer of [Datasets](../model/terminology.md#dataset) using a wire protocol. Both participants run Control and Data Planes.
+To put the document into the right context, some non-normative descriptions of the core concepts follow in this subsection.
 
-It is important to note that the Control and Data Planes are logical constructs. Implementations may choose to deploy both components within a single process or across heterogeneous clusters.
+#### 1.1.1 Control and Data Planes
 
-### 1.2 Dataset Transfer Types
+A TP involves two logical constructs, a control plane and a data plane. Services on the _control plane_ serve as a coordinating layer that receives messages and manages the local state of the TP (same as for the [Catalog Protocol](../model/terminology.md#catalog-protocol) and the [Contract Negotiation Protocol](../model/terminology.md#contract-negotiation-protocol)). The _data plane_ performs the actual transfer of data using a wire protocol. Both participants in a data sharing scenario run services located on control and/or data plane.
```

Comment:
```suggestion
A TP involves two logical constructs, a control plane and a data plane. Services on the _control plane_ are defined by this protocol and together serve as a coordinating layer that receive messages and manage the local state of the TP (same as for the [Catalog Protocol](../model/terminology.md#catalog-protocol) and the [Contract Negotiation Protocol](../model/terminology.md#contract-negotiation-protocol)). The _data plane_ performs the actual transfer of data using a wire protocol. Both participants in a data sharing scenario run services logically regarded as control and/or data plane services.
```

Reply:
Services on the control plane are not defined by this protocol... I tried to get away from any architectural patterns implying that control plane and data plane are services "inside" a connector.

Singular form was correct, referring to the coordination layer. I have rearranged the sentence and applied the other feedback.